File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        String answer = "";
        String front = phone_number.substring(0, phone_number.length() - 4);
        front = front.replaceAll(".", "*");
        String back = phone_number.substring(phone_number.length() - 4, phone_number.length());
        answer = front + back;
        return answer;
    }
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.swift
func solution(_ phone_number:String) -> String {
    var result: String = ""
    
    var len = phone_number.count
    var cnt: Int = 0
    for i in phone_number {
        if cnt < len - 4 {
            result += "*"
        } else {
            result += String(i)
        }
        cnt += 1
    }
    
    return result
}